site/en/docs/devtools/dom/index.md

```diff
@@ -176,6 +176,36 @@ To edit a node's type, double-click the type and then type in the new type.
 
    {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/nbdyNWefo9fqESvfWdha.png", alt="Changing the node type to button", width="800", height="545" %}
 
+### Edit as HTML {: #as-html }
+
+To edit nodes as HTML with syntax highlighting and autocomplete, select **Edit as HTML** from the node's drop-down menu.
+
+1. Right-click **Leonard** below and select **Inspect**.
+
+     * Howard
+     * Rajesh
+     * Leonard
```

Comment:
My only nit is add some female names. I realize this example and the next example are all male names.

Reply:
Like all lists in this guide, this is a pop culture reference. But sure, there's a woman there. I'll add her.